package.json

```diff
@@ -31,6 +31,9 @@
   },
   "dependencies": {
     "@cdssnc/webpack-starter": "^2.0.2",
+    "@csstools/postcss-sass": "^4.0.0",
+    "@fullhuman/postcss-purgecss": "^1.3.0",
+    "axios": "^0.19.0",
```

Comment:
Did you add axios?

I'm not seeing it used in the PR, and I'm not seeing the previous version in package.json

Reply:
was from a previous iteration of the code ... removed.